### PR TITLE
AKO 1.11.1 documentation: v1alpha1 to v1beta1. Changelog.

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
@abhishekbsingh @swathinsankaran @DixitAakash @arihantg  @aaha97 @saurabhvaidvmware: Please review this PR. In case of any issue(customer issues) fixed by you as part of AKO 1.11.1 is not mentioned, please mention here.

@DixitAakash : Please mention known issue here about SSORule.
@saurabhvaidvmware : Your PR is not merged so I haven't mentioned that issue.

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Review by apalsule — APPROVED** [review]

**Comment by saurabhvaidvmware** [comment]
@akshayhavile As the hostrule listener PR is merged now, can it also be added to this? Thanks

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by DixitAakash** [comment]
skip jira-id-check

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
@akshayhavile I have pushed the changes for SSORules in known issues. Can you please review the same.

**Comment by DixitAakash** [comment]
build ako

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

File: CHANGELOG.md [hunk 1]
```diff
@@ -405,3 +405,29 @@ All notable changes to this project will be documented in this file. The format
...
+ - Reduce the length of the label created by the AKO when the AutoFQDN is set as flat for service of Type LB.
+ - Do not add gslbFqdn, mentioned in Hostrule, to the vsvip fqdn when AKO is running in Dedicated mode.
+
+### Fixed
+ - Fix: AKO programs routes with secondary IPs in Openshift.
+ - Fix: When HTTPRules are applied to the ingresses, with same FQDN and path but deployed in different namespaces, then httprule config is removed from one of the pool when another httprule is applied.
+ - Fix: VS and Pools are not deleted when multiple ingress use same infrasetting for shard size.
+ - Fix: DNS mentioned in `external-dns.alpha.kubernetes.io/hostname` is not added as part `host-fqdn-vs-uuid-map` or vsvip dns info if `ako.vmware.com/enable-shared-vip` annotation is also applied to the LB service.
+ - Fix: Dedicated insecure VS gets both port `80` and `443` by default when no listeners are defined via any CRD.
+ - Fix: If only `loadBalancerIP` is given and no `listeners` are specified in the HostRule `tcpSettings` section, the VS creation fails with AKO log `error: Service mandatory for Virtual Service`.
```
Comment (akshayhavile) [inline CHANGELOG.md:430]: remove `AKO log`

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: CHANGELOG.md (hunk 1 above)
Comment (saurabhvaidvmware) [reply CHANGELOG.md]: Done

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Review by arihantg — APPROVED** [review]

**Review by DixitAakash — APPROVED** [review]

**Pull request merged**